### PR TITLE
Developing

### PR DESCRIPTION
**Pull request by Undistraction**
Please see individual commits

### CODE REVIEW

**Comment by Undistraction** [comment]
Hold on. Forgot the text linting

File: manuscript/05_Developing.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 -# Developing
 
-In this part, you get up and running with webpack. You learn to configure webpack-dev-server, Hot Module Replacement and, linting. Finally, you compose the configuration so that it's possible to expand in the following parts of the book.
+In this part, you get up and running with webpack. You learn to configure webpack-dev-server, Hot Module Replacement and linting. Finally, you compose the configuration so that it's possible to expand in the following parts of the book.
```
Comment (bebraw) [inline manuscript/05_Developing.md]: Oxford comma (`Replacement, and`). Better add a rule for that.

Comment (sapegin) [inline manuscript/05_Developing.md]: Inverted Oxford comma — after “and”? :-)

Comment (bebraw) [inline manuscript/05_Developing.md]: I made a mistake there, yup.

**Pull request merged**